pkg/apis/helm.bitnami.com/v1/types.go

```diff
@@ -24,10 +25,17 @@ type HelmReleaseSpec struct {
 	ChartName string `json:"chartName,omitempty"`
 	// Version is the chart version
 	Version string `json:"version,omitempty"`
+	// Auth is the authentication
+	Auth *HelmReleaseAuth `json:"auth,omitempty"`
```

Comment:
see comment in https://github.com/kubeapps/kubeapps/pull/247#discussion_r183001474 about not using a pointer here

Reply:
agreed.  I'd make this inline, and keep `Header` in `HelmReleaseAuth` a pointer (`HelmReleaseAuth` is an enum, so it's normal to use pointers to ensure that only one of the options is defined)